src/kubectl-helm-minikube/install.sh

```diff
@@ -134,6 +141,15 @@ if [ ${KUBECTL_VERSION} != "none" ]; then
         KUBECTL_VERSION="v${KUBECTL_VERSION}"
     fi
     curl -sSL -o /usr/local/bin/kubectl "https://dl.k8s.io/release/${KUBECTL_VERSION}/bin/linux/${architecture}/kubectl"
+    if [ -e "/usr/local/bin/kubectl" ]; then
+        if grep -q "The specified key does not exist." "/usr/local/bin/kubectl"; then 
+            repo_url=https://api.github.com/repos/kubernetes/kubernetes/releases
+            requested_version=$(get_previous_version "${repo_url}")
+            echo -e "\nAttempting to install ${requested_version}"
+            curl -sSL -o /usr/local/bin/kubectl "https://dl.k8s.io/release/${requested_version}/bin/linux/${architecture}/kubectl"
```

Comment:
Looks like we are using `curl -L -s https://dl.k8s.io/release/stable.txt` to find the latest version (this will always work). Hence, we don't need to add retry logic for `kubectl`. Can you clean it up?

Reply:
removed !